core/templates/pages/library-page/search-results/activity-tiles-infinity-grid.directive.ts

```diff
@@ -50,6 +53,9 @@ angular.module('oppia').directive('activityTilesInfinityGrid', [
             }
           };
           ctrl.$onInit = function() {
+            LoaderService.getLoadingMessageSubject().subscribe(
```

Comment:
Any reason for why this is being added here and not in showMoreActivities funciton?

Reply:
Adding this subscription in showMoreActivities means that it will create a new subscription every time the function is called. We only have to subscribe to the subject once to get the update from the subject.

Reply:
Right!